Text-Hoborg/lib/Text/Hoborg/Manuel.pod

```diff
@@ -234,7 +234,7 @@ Lost you all? Big, roundish, thick-on-the-bottom, metallic, upside-down
 bucket with another on top, body and head. Which goes to account for
 the name mechanical man, but when I first saw him that what I though
 about. It was even steaming a little bit, because he's not actually
-mechanical, you see, it's also fuel-powered ad some electrickal cables
+mechanical, you see, it's also fuel-powered and has some electrical cables
 to spark it to motion whenever it's needed. Or something like that. I
```

Comment:
Once again, "electrickal" is intended and is probably closer to the initial spelling of the word. The other is obviously an error, thanks for pointing it out.